packages/node/test/manual/release-health/session-aggregates/aggregates-disable-single-session.js

```diff
@@ -12,6 +12,9 @@ function cleanUpAndExitSuccessfully() {
 }
 
 function assertSessionAggregates(session, expected) {
+  if (!session.aggregates) {
```

Comment:
this already failed before, but did not result in test failures because apparently some global event processors were not running and thus no sentry error was produced for this (?)